apps/prairielearn/src/lib/preact-html.ts

```diff
@@ -24,5 +24,9 @@ export function renderHtml(vnode: VNode): HtmlSafeString {
     pretty = true;
   }
 
+  if (typeof vnode === 'string') {
+    return unsafeHtml(vnode);
```

Comment:
This scares me quite a bit; it seems like this would make it really really easy to accidentally include unescaped data onto a page.

I believe this change was made in response to a comment from me suggesting that components could just return strings (which is a normal thing for React/Preact components to do). I think that probably a more correct thing to do is to do `return html(vnode)` here to ensure that things *are* always escaped.

Reply:
Changed to:

```
  if (typeof vnode === 'string') {
    return escapeHtml(new HtmlSafeString([vnode], []));
  }
```